src/rest.js

```diff
@@ -37,9 +37,14 @@ function find(config, auth, className, restWhere, restOptions, clientSDK) {
 
 // get is just like find but only queries an objectId.
 const get = (config, auth, className, objectId, restOptions, clientSDK) => {
+  var restWhere = { objectId };
   enforceRoleSecurity('get', className, auth);
-  const query = new RestQuery(config, auth, className, { objectId }, restOptions, clientSDK);
-  return query.execute();
+  return triggers.maybeRunQueryTrigger(triggers.Types.beforeFind, className, restWhere, restOptions, config, auth).then((result) => {
```

Comment:
I think it would be useful but it will be annoying. We don't have a safe way to be 100% sure that the frontend is trying to do a get and not a find. The js SDK uses the find endpoint for getting (https://github.com/parse-community/Parse-SDK-JS/blob/master/src/ParseQuery.js#L284). That's why this test (https://github.com/parse-community/parse-server/blob/master/spec/CloudCode.spec.js#L117) already works in the master without this change but this new one (https://github.com/parse-community/parse-server/pull/3865/files#diff-57110aba585acf52c5d9209a61443a8eR1309) doesnt. Therefore I think the best way for someone to check if it is a get or a find is by checking if the where query contains only objectId among the keys.